graphql/integration_atomic_test.go

```diff
@@ -55,6 +56,8 @@ func setup(t *testing.T, directory string) atomicGraphQLState {
 	env := evergreen.GetEnvironment()
 	ctx := context.Background()
 	require.NoError(t, env.DB().Drop(ctx))
+	logsDb := env.Client().Database(model.TaskLogDB)
```

Comment:
I would add the logs db name to the state struct (and access it in the setupData function)